spec/features/authentication_spec.rb

```diff
@@ -32,6 +32,47 @@
         expect(page).to have_content(I18n.t('sessions.create.failure'))
       end
     end
+
+    context 'with no authentication token' do
+      let(:request_for_comment) { create(:rfc_with_comment, user: user) }
+
+      it 'denies access to the request for comment' do
+        visit(request_for_comment_path(request_for_comment))
+        expect(page).to have_content(I18n.t('application.not_authorized'))
```

Comment:
We could check here for the URLand the RfC title as well:
```
expect(current_url).not_to be(rfc_link)
expect(page).not_to have_content(request_for_comment.exercise.title)
```

Reply:
Using a `response` in feature specs is neither supported nor recommended (as this should be done in a request spec). Therefore, we could not implement the spec as suggested here, but rather sticked with the existing method to verify the URL and the exercise title.